swerve100/src/main/java/frc/robot/subsystems/AHRSClass.java

```diff
@@ -0,0 +1,174 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.I2C;
+import edu.wpi.first.wpilibj.SerialPort;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/** Add your docs here. */
+public class AHRSClass implements Sendable {
+    public static boolean gyrosWorking = true;
+    public final AHRS m_gyro1;
+    public final AHRS m_gyro2;
+    public static boolean gyro1Connected = true;
+    public static boolean gyro2Connected = true;
+    float gyroZOffset_I2C;
+    float gyroZOffset_USB;
+    public AHRSClass() {
+        m_gyro1 = new AHRS(SerialPort.Port.kUSB);
+        m_gyro2 = new AHRS(I2C.Port.kMXP);
+        m_gyro1.enableBoardlevelYawReset(true);
+        m_gyro2.enableBoardlevelYawReset(true);
+        // while (m_gyro2.isCalibrating() || m_gyro1.isCalibrating()){
```

Comment:
i think you can't just comment out the calibration, you need to figure out a way to wait for the gyros that exist.

Reply:
i think this still needs attention.  maybe just add something about being connected, e.g. "while (gyro1.isConnected && gyro1.iscalibrating || gyro2.isConnected && gyro2.isCalibrating)"